lib/dao.js

```diff
@@ -1242,21 +1243,21 @@ DataAccessObject._normalize = function(filter) {
   }
   var err = null;
   if ((typeof filter !== 'object') || Array.isArray(filter)) {
-    err = new Error(util.format('The query filter %j is not an object', filter));
+    err = new Error(g.f('The query filter %j is not an object', filter));
     err.statusCode = 400;
     throw err;
   }
   if (filter.limit || filter.skip || filter.offset) {
     var limit = Number(filter.limit || 100);
     var offset = Number(filter.skip || filter.offset || 0);
     if (isNaN(limit) || limit <= 0 || Math.ceil(limit) !== limit) {
-      err = new Error(util.format('The limit parameter %j is not valid',
+      err = new Error(g.f('The limit parameter %j is not valid',
         filter.limit));
       err.statusCode = 400;
       throw err;
     }
     if (isNaN(offset) || offset < 0 || Math.ceil(offset) !== offset) {
-      err = new Error(util.format('The offset/skip parameter %j is not valid',
+      err = new Error(g.f('The {{offset/skip}} parameter %j is not valid',
```

Comment:
Does `g.f()` contains the logic of `util.format` inside? if not I think we'd better keep it as `g.f(util.format())`?